<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music_homeClass</class>
 <widget class="QWidget" name="music_homeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>music_home</string>
  </property>
  <widget class="QLabel" name="label_music_name">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>320</y>
     <width>70</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>null.mp3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>载入歌曲</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_jump">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>31</width>
     <height>160</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_jump_name">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>跳跃幅度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jump">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="music_home.qrc"/>
 </resources>
 <connections/>
</ui>
